samples/mcuboot_config/mcuboot_config.template.h

```diff
@@ -116,4 +116,17 @@
  * "assert" is used. */
 /* #define MCUBOOT_HAVE_ASSERT_H */
 
+/*
+ * Watchdog tickling
```

Comment:
The conventional term for this is "kicking" the watchdog. The typical more polite term is "petting" the watchdog. Tickling makes for an interesting metaphor.

Reply:
We use term `feed` in Nordic nrfx watchdog driver.

Reply:
For what it's worth, I've heard "tickle" used before.